src/libraries/System.Net.Security/ref/System.Net.Security.cs

```diff
@@ -166,6 +147,13 @@ public partial class SslClientAuthenticationOptions
         public System.Net.Security.RemoteCertificateValidationCallback? RemoteCertificateValidationCallback { get { throw null; } set { } }
         public string? TargetHost { get { throw null; } set { } }
     }
+    public readonly partial struct SslClientHelloInfo
+    {
+        private readonly object _dummy;
+        private readonly int _dummyPrimitive;
```

Comment:
Note this could cause some existing compiling code to stop compiling. (But we should still fix it.)